build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
I tried 384.47 with 381.22, it worked. Though I am not sure if that is the case with all versions